### PR TITLE
Update CHANGES with info about AuthEnvelopedData addition

### PR DESCRIPTION
**Pull request by bukka**
Update of CHANGES.md for https://github.com/openssl/openssl/pull/8024 .

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -437,6 +437,10 @@ OpenSSL 3.0
 
    *Antonio Iacono*
 
+ * Added the AuthEnvelopedData (RFC 5083) with AES-GCM parameter (RFC 5084).
```
Comment (t8m) [inline CHANGES.md:440]: Could you  please expand the text a little? Add something like `Added ... support to the CMS implementation.` Or something like that to make it clear without having to look into the RFCs or code where the support was added.

Comment (bukka) [reply CHANGES.md]: @t8m Makes sense. I have extended the text a bit. Please let if you want me to change anything in it.

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
@paulidale Please reapprove

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master, thank you.

**Pull request closed without merging**